src/components/trip_view/cal/event_modal/EventModalContainer.js

```diff
@@ -0,0 +1,20 @@
+import { connect } from 'react-redux';
+import { EventModal } from './EventModal';
+import { updatePlanInDb } from '../../../../redux/actions/planActions';
+
+const mapStateToProps = (state) => {};
+
+const mapDispatchToProps = (dispatch) => {
+  return {
+    onUpdatePlan: (plan) => {
+      dispatch(updatePlanInDb(plan));
```

Comment:
Does this only work with plans? Or does it also work with other event types?

Reply:
This shouldn't be here anymore.